data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/schema/GetSyncCommitteeRewards.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title" : "GetSyncCommitteeRewards",
+  "type" : "object",
+  "required" : [ "execution_optimistic", "finalized", "data" ],
+  "properties" : {
+    "execution_optimistic" : {
+      "type" : "boolean"
+    },
+    "finalized" : {
+      "type" : "boolean"
+    },
+    "data" : {
+      "type" : "array",
+      "items" : {
+        "type" : "object",
+        "required" : [ "validator_index", "reward" ],
+        "properties" : {
+          "validator_index" : {
+            "type" : "string",
+            "description" : "integer string",
```

Comment:
These descriptions are the only real information the consumer has, lets make it something like `The validator index receiving this reward`
and maybe `The sync committee reward in GWEI for the validator` for the reward value...